checked_yaml/example/example.dart

```diff
@@ -36,10 +36,18 @@ class Configuration {
 }
 
 void main(List<String> arguments) {
-  final fileContents = File(arguments.single).readAsStringSync();
+  var sourceUrl = arguments.single;
```

Comment:
nit: it looks like this is either a source url or a yaml string, consider renaming it if so? `sourceUrlOrYaml` or something.

Or, just drop support for one or the other, I think it distracts from the example.

Reply:
It's kinda nice to support either – at least it makes my tests work.

Will rename.

Reply:
fixed!